apps/earn-protocol/hooks/use-tokens-balances.ts

```
@@ -39,13 +39,15 @@ export const useTokenBalances = ({
 
   const arbitrumTokenBalance = useTokenBalance({
     tokenSymbol,
+    vaultTokenSymbol: tokenSymbol,
```

Biome (1.9.4)

[error] This property value named vaultTokenSymbol is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named vaultTokenSymbol


(lint/suspicious/noDuplicateObjectKeys)